networkx/algorithms/matching.py

```diff
@@ -829,3 +837,38 @@ def verifyOptimum():
         verifyOptimum()
 
     return mate
+
+
+def is_max_cardinality_matching(G, matching):
```

Comment:
I don't think this function is valuable, as it just computes a maximum cardinality matching anyway. See the same comment on pull request #1915.